shared/actions/platform-specific/index.desktop.js

```diff
@@ -242,14 +245,31 @@ const sendKBServiceCheck = (state: TypedState, action: ConfigGen.DaemonHandshake
   }
 }
 
+const checkOutOfDate = () =>
```

Comment:
Let's name this something like startOutOfDateCheckLoop or something to make it clear it runs forever